website/app/templates/patterns.hbs

```diff
@@ -3,22 +3,30 @@
 <Doc::Page::Stage>
   <Doc::Page::Cover @title="Patterns" @description="Coming soon" />
   <Doc::Page::Content>
-    <h2 class="doc-text-h2">What is a pattern?</h2>
-    <p class="doc-text-body">A pattern is a combination of components that work together to accomplish a function or
-      render a layout. They are often used to solve the same problem consistently. Common examples of patterns are
-      forms, navigation, button sets, etc.</p>
-    <h2 class="doc-text-h2">Helios pattern support</h2>
-    <p class="doc-text-body">We don't currently support patterns, but we have a few on our
-      <a href="https://go.hashi.co/hds-rollout">roadmap</a>. We're also planning to introduce guidelines and best
-      practices around assembling components in a consistent and scalable manner.
+    <div class="doc-banner doc-banner--type-insight">
+      <FlightIcon class="doc-banner__icon" @name="bulb" @size="24" @isInlineBlock="false" />
+      <div class="doc-banner__content">
+        <p class="doc-markdown-p"><strong>What is a pattern?</strong></p>
+        <p class="doc-markdown-p">A pattern is a combination of components that work together to accomplish a function
+          or render a layout. They are often used to solve the same problem consistently. Common examples of patterns
+          are forms, navigation, button sets, etc.</p>
+      </div>
+    </div>
+    <h2 class="doc-text-h2">Patterns in Helios</h2>
+    <p class="doc-text-body">We've spent much of the past year building lower level components, and therefore, don't yet
+      offer patterns. See which patterns we plan to work on in the
+      <a href="https://go.hashi.co/hds-rollout">Helios roadmap</a>.
```

Comment:
@jorytindall can you open a follow-up ticket and add the `doc-link-generic` class to the links, so they will use the right colors? thanks

FYI this will be automated later, see https://hashicorp.atlassian.net/browse/HDS-1354